packages/api/migrations/1_rename-boba-networks.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+    async up(db) {
+        return Promise.all(
+            (await db.collection("result_request").find({
+                feedFullName: /boba-/
+            }))
+            .map(result => {
+                return {
+                    ...result,
+                    feedFullName: result.feedFullName.replace(
+                        /boba-/, 
+                        "boba-ethereum-"
+                    )
+                }
+            })
+            .map(result => {
+                return db.updateOne(
+                    { _id: result._id },
+                    { $set: { feedFullName: result.feedFullName }}
+                )
+            })
+        )
+    }
```

Comment:
I had to do some small changes to make it work:

```suggestion
    const resultRequestCollection = db.collection('result_request')

    const promises = (
      await resultRequestCollection
        .find({
          feedFullName: /boba-/
        })
        .toArray()
    )
      .map(resultRequest => ({
        ...resultRequest,
        feedFullName: resultRequest.feedFullName.replace(
          /boba-/,
          'boba-ethereum-'
        )
      }))
      .map(resultRequest =>
        resultRequestCollection.updateOne(
          { _id: resultRequest._id },
          { $set: { feedFullName: resultRequest.feedFullName } }
        )
      )

    await Promise.all(promises)
```